crates/divan_compat/divan_fork/src/divan.rs

```diff
@@ -310,6 +310,32 @@ impl Divan {
 
                 if should_compute_stats {
                     let stats = bench_context.compute_stats();
+                    {
+                        let name = bench_entry.display_name().to_string();
+                        let file = bench_entry.meta().location.file;
+                        let module_path = bench_entry
+                            .meta()
+                            .module_path_components()
```

Comment:
on the instrumented side I use directly entry.meta.module_path, should I change it to be the same as here ?

(I also noted the changes with the `::` which I will incorporate)

Reply:
ah good catch. Initially I used this to skip the first component of the module path(since it's the current file mod name, so basically the filename without `.rs`. I ended up putting it back in the module path but I think this shouldn't be here